website/docs/components/link/standalone/partials/guidelines/guidelines.md

```diff
@@ -52,41 +52,84 @@ A Standalone Link can include a leading or trailing icon. Avoid creating links w
 In most cases, use leading icons. Choose icons that add meaning and clarity to the action described in the link’s text.
 
 <Doc::Layout @spacing="12px">
-  <Hds::Link::Standalone @text="Deploy with Terraform" @icon="zap" @iconPosition="leading" @color="primary" @route="components" />
   <Hds::Link::Standalone @text="Download" @icon="download" @iconPosition="leading" @color="primary" @route="components" />
   <Hds::Link::Standalone @text="View connections" @icon="network-alt" @iconPosition="leading" @color="primary" @route="components" />
 </Doc::Layout>
 
 <Doc::Layout @spacing="12px">
-  <Hds::Link::Standalone @text="Deploy with Terraform" @icon="zap" @iconPosition="leading" @color="secondary" @route="components" />
   <Hds::Link::Standalone @text="Download" @icon="download" @iconPosition="leading" @color="secondary" @route="components" />
   <Hds::Link::Standalone @text="View connections" @icon="network-alt" @iconPosition="leading" @color="secondary" @route="components" />
 </Doc::Layout>
 
-Use the leading position when using service icons (e.g., GitHub).
+!!! Do
+
+Consider directionality; if directing the user backward in a flow, use the leading position. 
+
+<Doc::Layout @spacing="12px">
+  <Hds::Link::Standalone @text="Back to Dashboard" @icon="chevron-left" @iconPosition="leading" @color="primary" @route="components" />
+  <Hds::Link::Standalone @text="Back to Dashboard" @icon="chevron-left" @iconPosition="leading" @color="secondary" @route="components" />
+</Doc::Layout>
```

Comment:
I don't think we need to duplicate every time the example for primary and secondary color (it was confusing, for me). if you want to show that there are different variants, alternate the examples between the two colors

(this applies to other examples too)

```suggestion
<Hds::Link::Standalone @text="Back to Dashboard" @icon="chevron-left" @iconPosition="leading" @color="primary" @route="components" />
```